### ISSUE TITLE
False wall's don't update their textures properly

### ISSUE THREAD

**Issue by monkeymilesmoo** [goonstation/goonstation#14517]
### Labels

[BUG] [MINOR]

### Description

If you break a wall next to a false wall it won't update its texture, even if you place another wall next to it. The only way to actually update the texture is to place a false wall next to the existing false wall.

### Steps to reproduce

1. Place false wall next to a normal wall
2. Break normal wall
3. False wall still points to where normal wall was
4. Place second false wall in a different spot adjacent to first false wall
5. First false wall finally updates texture

### Expected Behavior

False walls shouldn't give themselves away by having a janky texture

**Comment by flappybatpal** [comment]
From testing false walls ONLY update when another false wall is built next to them.

**Comment by monkeymilesmoo** [comment]
Yes that's the bug, I think it's because the false wall has a slightly different path and when placing walls they only update their own type, so false walls update false walls and walls update walls
